bin/lesson_initialize.py

```diff
@@ -47,5 +47,11 @@ def main():
         )
 
 
+if __name__ == '__main__':
```

Comment:
This whole section doesn't look right, I think there's a copy/paste error here

Reply:
Same issue resolved for hpc-intro resolved in https://github.com/hpc-carpentry/hpc-intro/pull/207

Reply:
Fixed it here so this is good to go